drivers/usb/device/usb_dc_stm32.c

```diff
@@ -254,6 +254,25 @@ static int usb_dc_stm32_clock_enable(void)
 		LOG_ERR("Unable to set USB clock source to PLL.");
 	}
 #endif /* CONFIG_CLOCK_STM32_MSI_PLL_MODE && !CONFIG_CLOCK_STM32_SYSCLK_SRC_MSI */
+
+#elif defined(RCC_CFGR_OTGFSPRE) \
+	&& defined(LL_RCC_USB_CLKSOURCE_PLL_DIV_2) \
```

Comment:
`LL_RCC_USB_CLKSOURCE_PLL_DIV_2` and `LL_RCC_USB_CLKSOURCE_PLL_DIV_3` defined under `RCC_CFGR_OTGFSPRE`.
So first line should be enough